scout/adapter/mongo/variant.py

```diff
@@ -334,85 +334,6 @@ def overlapping_sv_variant(self, case_id, variant_obj):
             if hit["length"] == closest_length:
                 return hit
 
-    def gene_variants(
```

Comment:
I've removed all this code, and replaced it with a direct query in the gene_variant view